lease/lessor.go

```diff
@@ -50,6 +50,9 @@ var (
 	// maximum number of lease checkpoints to batch into a single consensus log entry
 	maxLeaseCheckpointBatchSize = 1000
 
+	// interval to check if the expired lease is revoked
+	expiredleaseRetryInterval = 3 * time.Second
```

Comment:
Do we know what is the main reason for revoke failure? If it is due to revoke request timeout, we probably want to make `expiredleaseRetryInterval` longer than the default request timeout:
https://github.com/etcd-io/etcd/blob/e3f37534e1f9d0f8a9094e70eb031d484694f40f/etcdserver/v3_server.go#L598